.changeset/tasty-masks-kiss.md

```diff
@@ -2,4 +2,4 @@
 "@burnt-labs/abstraxion": minor
 ---
 
-Add grantee signer client to seamlessly handle grantor/grantee relationships
+Add grantee signer client to seamlessly handle granter/grantee relationships
```

Comment:
Does the renaming change any of the public api? If so, we should add another changeset for it.